### DISCUSSION TITLE
Show the number of sessions

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1291] [Anything Else]
"We are a news and information website dealing in the B2B market – an online magazine, if you like. From our point of view, the number of sessions is the most important metric. Even if a user visits to read the same news story more than once that is just an indication of their level of interest."

(request via email)

this hasn't come up very frequently but came up twice in a week now. we already do session based tracking in the background so could be nice to surface the number of sessions metric somewhere. perhaps if you click on "unique visitors" it switches to "number of sessions" or something along those lines

**Update:** You can now find two brand new metrics in your dashboard: "Total visits" (sessions) and "Views per visit" (pageviews / session).

**Comment by metmarkosaric** [comment]
"we’re looking for a metric similar to the Google session for our performance reporting. Our intention is to use Plausible for performance reporting on metrics where coverage is most important, supplemented by GA for behavioural analytics. To do this it would be great to have an equivalent for sessions in Plausible;

* Because sessions, sessions per user and ave time per session are very useful metrics for measuring changes in engagement over time (and are included in our core KPI reporting)
* To track the difference between GA and Plausible on the two main GA traffic concepts (users and sessions) so we can use both platforms to build overall analyses of behaviour."

**Comment by RobertJoonas** [comment]
While the number of sessions is not visible on the dashboard yet, it would be great to add it to the Stats API without any UI work.

**Reply by akweb** [reply]
It would be really great to be able to query / view sessions.

**Reply by RobertJoonas** [reply]
The number of sessions is now available from the Stats API (Aggregate, Timeseries and Breakdown endpoints). You can specify `metrics=visits` in your query to get the number of sessions. See https://plausible.io/docs/stats-api

**Reply by akweb** [reply]
Wow, that's awesome. Thanks a lot! We'll be checking it out this week.

**Comment by fabbow1337** [comment]
That feature would add lots of value while being fairly easy to implement if you just have to surface it. +1!

**Comment by mgntrn** [comment]
I would love to view 'visits' next to 'unique visitors' in the dashboard as well! Glad to see that I can get it from the API though.

**Comment by metmarkosaric** [comment]
**Update**: You can now find two brand new metrics in your dashboard: "Total visits" (sessions) and "Views per visit" (pageviews / session).

**Reply by mgntrn** [reply]
Awesome! I'm using the self-hosted version, so I guess I should wait a little longer, but great work, thanks!

**Comment by akweb** [comment]
Hi there. Is there any ETA on when we will see this in the self hosted version?
Thank you, Andreas

**Reply by ruslandoga** [reply]
@akweb 👋

I'm already working on release notes for the next release :)

**Reply by ruslandoga** [reply]
I'll use https://github.com/plausible/hosting/issues/68 to track progress
